<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab10_PNR</class>
 <widget class="QMainWindow" name="Lab10_PNR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab10_PNR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit1_PNR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblNumerator_PNR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Numerator:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDenominator_PNR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Denominator:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNum1_PNR">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNum2_PNR">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl1_1_PNR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl1_2_PNR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDenom1_PNR">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl2_2_PNR">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl2_1_PNR">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDenom2_PNR">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd_PNR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSub_PNR">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Subtract</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMult_PNR">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Multiply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDiv_PNR">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>330</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Divide</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMsg_PNR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Message:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMsgVal_PNR">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
